pilot/cmd/pilot-discovery/main.go

```diff
@@ -130,6 +130,10 @@ func init() {
 	discoveryCmd.PersistentFlags().StringVarP(&serverArgs.Config.ControllerOptions.WatchedNamespace, "appNamespace",
 		"a", metav1.NamespaceAll,
 		"Restrict the applications namespace the controller manages; if not set, controller watches all namespaces")
+	discoveryCmd.PersistentFlags().MarkDeprecated("appNamespace",
```

Comment:
Probably ok - I would have just reused the flag and 'extended' it to support comma separate list.

Reply:
(given that this all is going away in master anyways - too much trouble for an "s")

Reply:
Renamed `appNamespaces` to `appNamespace` and keep reusing old parameter.